src/views/refund/refund.js

```diff
@@ -24,6 +24,16 @@ const styles = theme => ({
   },
 });
 
+const uploadRefundFileText = (refundFile, txHash) => {
+  if (Object.keys(refundFile).length === 0) {
+    return 'Upload refund file';
+  } else if (txHash === '') {
+    return 'Input transaction hash';
+  } else {
+    return 'Upload refund file';
```

Comment:
With the changes in the last commit we can get rid of the `} else {` because all cases are already covered with the two statements above.